.travis.yml

```diff
@@ -2,7 +2,9 @@ language: java
 jdk:
   - oraclejdk8
 addons:
-  sauce_connect: true
+  sauce_connect:
+    username: alloyui
```

Comment:
@fernandosouza I was just curious why you added the username if the issue was not related to that. Could you check if it works without the username, to keep it cleaner? Thanks.